domain/src/main/java/org/fao/geonet/repository/UserSavedSelectionRepository.java

```diff
@@ -34,4 +36,7 @@ public interface UserSavedSelectionRepository extends
     GeonetRepository<UserSavedSelection, UserSavedSelectionId>,
     JpaSpecificationExecutor<UserSavedSelection>,
     UserSavedSelectionRepositoryCustom {
+
+    @Query("SELECT COUNT(DISTINCT u.user.id) FROM UserSavedSelection u WHERE u.metadataUuid = (:uuid)")
```

Comment:
It's hard to call what is popularity.
IMO, it would be better to know for each selection id rather than for all selections.
A `group by` and an object in the index might make sense ?
Or at least use the `preferred records` list ? `selectionId = 0`

Reply:
I guess we can go with only the saves in id=0, in most instances that makes sense (id=0 is "preferred records")